pymemcache/client/base.py

```diff
@@ -864,7 +872,7 @@ def __init__(self,
 
     def check_key(self, key):
         """Checks key and add key_prefix."""
-        return _check_key(key, key_prefix=self.key_prefix)
+        return _check_key(key, unicode_key_support=self.unicode_key_support, key_prefix=self.key_prefix, )
```

Comment:
Maybe trim this trailing comma since this convention has not been kept in the rest of the repo